src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -389,6 +389,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                     recursively when uploading from local file system.')
         c.argument('put_md5', action='store_true', help='Create an MD5 hash of each file, and save the hash \
                     as the Content-MD5 property of the destination blob/file.Only available when uploading.')
+        c.argument('blob_type', arg_type=get_enum_type(["BlockBlob", "PageBlob", "AppendBlob"]),
```

Comment:
Is the enum type defined in Python SDK? 
E.g. https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/storage/azure-mgmt-storage/azure/mgmt/storage/v2019_04_01/models/_storage_management_client_enums.py

Reply:
No. Because it is not a popular type format, I just defined here.